pkg/configtx/application.go

```diff
@@ -66,14 +66,10 @@ func (c *ConfigTx) ApplicationConfiguration() (Application, error) {
 	}, nil
 }
 
-// AddAnchorPeer adds an anchor peer to an existing channel config transaction.
-// It must add the anchor peer to an existing org and the anchor peer must not already
-// exist in the org.
-func (c *ConfigTx) AddAnchorPeer(orgName string, newAnchorPeer Address) error {
-	applicationOrgGroup, ok := c.updated.ChannelGroup.Groups[ApplicationGroupKey].Groups[orgName]
-	if !ok {
-		return fmt.Errorf("application org %s does not exist in channel config", orgName)
-	}
+// SetAnchorPeer sets an anchor peer to an existing channel config transaction.
+// If anchor peer endpoints already exist in configuration, this action will be a no-op.
+func (c *ConfigTx) SetAnchorPeer(orgName string, newAnchorPeer Address) error {
```

Comment:
This is a tricky one ... I believe that technically we are **adding** and anchor peer rather than simply just **setting** one

Reply:
Depends on how you interpret the data. You could argue you're setting the existence of that anchor peer. I do agree by and large this is really just an add since we treat the Anchor Peers data as a slice, but the idea was to attempt standardizing the terminology for add/update/set. Either way the comment should be updated if we decide to go with set since `SetAnchorPeer sets an anchor peer to an existing channel` doesn't exactly sound correct 😛I'm fine keeping this as Add if there's agreement on it.